tests/suites/test_suite_cipher.aes.data

```diff
@@ -4,7 +4,7 @@ dec_empty_buf:MBEDTLS_CIPHER_AES_128_CBC:0:0
 
 AES-128 XTS - Decrypt empty buffer
 depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_XTS
-dec_empty_buf:MBEDTLS_CIPHER_AES_128_XTS:MBEDTLS_ERR_AES_INVALID_INPUT_LENGTH:0
+dec_empty_buf:MBEDTLS_CIPHER_AES_128_XTS:-0x22:0
```

Comment:
Where does this numerical value come from? Please use symbolic constant names. This test case depends on `MBEDTLS_AES_C` so `MBEDTLS_ERR_AES_INVALID_INPUT_LENGTH` is guaranteed to be defined.